content/en/docs/reference/glossary/cidr.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: CIDR
```

Comment:
More people know this as CIDR than know what the expanded term actually _means_, so the original works better for me.